pkgs/applications/office/ledger/default.nix

```diff
@@ -27,6 +27,10 @@ stdenv.mkDerivation rec {
     "-DCMAKE_INSTALL_LIBDIR=lib"
     "-DBUILD_DOCS:BOOL=ON"
     (lib.optionalString usePython "-DUSE_PYTHON=true")
+  ] ++ lib.optionals (usePython && stdenv.isDarwin) [
+    # Fix python lookup on Darwin. Not necessary after
+    # https://github.com/NixOS/nixpkgs/pull/94090 lands in master
```

Comment:
Is this comment correct? That PR has already landed in `master`, and I believe the `ledger` build failure I was seeing was with the latest `master`.

Reply:
it was reverted in https://github.com/NixOS/nixpkgs/commit/22f82fc8caa6d65e20324c237940e3a668d312e9, the offending line is still there: https://github.com/NixOS/nixpkgs/blob/e31a4679d502cb4ea07236eca8faba1ce1dc67b9/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L51
I thought it landed into `staging`, but doesn't seem to be there cc: @matthewbauer

Reply:
Aha, I see! Thanks.